tests/test_transforms/test_timestamp/test_special_days_transform.py

```diff
@@ -105,47 +104,47 @@ def test_interface_noweek_nomonth():
 
 
-def test_interface_two_segments_week(constant_days_two_segments_df: pd.DataFrame):
+def test_interface_two_segments_week(constant_days_two_segments_ts: TSDataset):
     """
     This test checks that SpecialDaysTransform that should find special weekdays creates the only column with
     'anomaly_weekdays' name as expected.
     """
     special_days_finder = SpecialDaysTransform(find_special_weekday=True, find_special_month_day=False)
-    df = special_days_finder.fit_transform(constant_days_two_segments_df)
+    df = special_days_finder.fit_transform(constant_days_two_segments_ts).to_pandas()
     for segment in df.columns.get_level_values("segment").unique():
         assert "anomaly_weekdays" in df[segment].columns
         assert "anomaly_monthdays" not in df[segment].columns
         assert df[segment]["anomaly_weekdays"].dtype == "category"
 
 
-def test_interface_two_segments_month(constant_days_two_segments_df: pd.DataFrame):
+def test_interface_two_segments_month(constant_days_two_segments_ts: TSDataset):
     """
     This test checks that SpecialDaysTransform that should find special month days creates the only column with
     'anomaly_monthdays' name as expected.
     """
     special_days_finder = SpecialDaysTransform(find_special_weekday=False, find_special_month_day=True)
-    df = special_days_finder.fit_transform(constant_days_two_segments_df)
+    df = special_days_finder.fit_transform(constant_days_two_segments_ts).to_pandas()
     for segment in df.columns.get_level_values("segment").unique():
         assert "anomaly_weekdays" not in df[segment].columns
         assert "anomaly_monthdays" in df[segment].columns
         assert df[segment]["anomaly_monthdays"].dtype == "category"
 
 
-def test_interface_two_segments_week_month(constant_days_two_segments_df: pd.DataFrame):
+def test_interface_two_segments_week_month(constant_days_two_segments_ts: TSDataset):
     """
     This test checks that SpecialDaysTransform that should find special month and week days
     creates two columns with 'anomaly_monthdays' and 'anomaly_weekdays' name as expected.
     """
     special_days_finder = SpecialDaysTransform(find_special_weekday=True, find_special_month_day=True)
-    df = special_days_finder.fit_transform(constant_days_two_segments_df)
+    df = special_days_finder.fit_transform(constant_days_two_segments_ts).to_pandas()
     for segment in df.columns.get_level_values("segment").unique():
         assert "anomaly_weekdays" in df[segment].columns
         assert "anomaly_monthdays" in df[segment].columns
         assert df[segment]["anomaly_weekdays"].dtype == "category"
         assert df[segment]["anomaly_monthdays"].dtype == "category"
 
 
-def test_interface_two_segments_noweek_nomonth(constant_days_two_segments_df: pd.DataFrame):
+def test_interface_two_segments_noweek_nomonth():
     """This test checks that bad-inited SpecialDaysTransform raises AssertionError during fit_transform."""
```

Comment:
Not very good description of the test, I can't see any `AssertionError`. (I understand that it was before this PR).